serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/servicerecord/ServiceRecordFactory.java

```diff
@@ -55,8 +55,18 @@ public ServiceRecordFactory(ServiceregistryProperties properties, VirkSertServic
     @PreAuthorize("#oauth2.hasScope('move/dpo.read')")
     public ServiceRecord createEduServiceRecord(String orgnr) {
         String finalOrgNumber = ksLookup.mapOrganizationNumber(orgnr);
-        Endpoint ep = elmaLookupService.lookup(NORWAY_PREFIX + finalOrgNumber);
         String pemCertificate = lookupPemCertificate(finalOrgNumber);
+
+        Endpoint ep = elmaLookupService.lookup(NORWAY_PREFIX + finalOrgNumber);
+        String adr = ep.getAddress().toString();
+        if (adr.contains("#")) {
+            String uri = adr.substring(0, adr.indexOf("#"));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Put single-quotes around '#' to use the faster "indexOf(char)" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS3027)